daemon/algod/api/algod.oas2.json

```diff
@@ -2530,6 +2669,40 @@
         }
       }
     },
+    "Box": {
+      "description": "Box name and its content.",
+      "type": "object",
+      "required": [
+        "name",
+        "value"
+      ],
+      "properties": {
+        "name": {
+          "description": "\\[name\\] box name, base64 encoded",
+          "type": "string",
+          "format": "byte"
+        },
+        "value": {
+          "description": "\\[value\\] box value, base64 encoded.",
+          "type": "string",
+          "format": "byte"
+        }
+      }
+    },
+    "BoxDescriptor": {
+      "description": "Box descriptor describes a Box.",
+      "type": "object",
+      "required": [
+        "name"
+      ],
+      "properties": {
+        "name": {
+          "description": "Base64 encoded box name",
+          "type": "string",
+          "format": "byte"
```

Comment:
I'm confused as to why we would both `Base64` encode a box name yet return raw bytes? Apologies if this has already been discussed and agreed upon.

@michaeldiamant @ahangsu

Reply:
@tzaffi Trying to clarify:  Are you asking why the line reads `"format": "byte"` rather than something base64-specific?

* If so, the question is addressed via https://github.com/OAI/OpenAPI-Specification/blob/main/versions/2.0.md#data-types.  The PR matches the spec for base64 encoded values.
* If I misinterpreted the question, can you provide a bit more context?

Reply:
Thanks for clarifying that these specifications are actually consistent with Swagger's notion of base 64 encoding. In that case, I have a question for @ahangsu. I have stepped through the rest client test and reached [this point of execution](https://github.com/algorand/go-algorand/blob/f32a94333167f6dd5a9cf5108539faeacc395fdb/daemon/algod/api/client/restClient.go#L441).

This is in the same call starting from the [e2e test step](https://github.com/algorand/go-algorand/blob/f32a94333167f6dd5a9cf5108539faeacc395fdb/daemon/algod/api/client/restClient.go#L441).

When inspecting the local variables I see the following:

```
response
github.com/algorand/go-algorand/daemon/algod/api/server/v2/generated.BoxResponse {Name: []uint8 len: 3, cap: 3, [102,111,111], Value: []uint8 len: 5, cap: 6, [98,97,114,49,50]}
Name:
[]uint8 len: 3, cap: 3, [102,111,111]
Value:
[]uint8 len: 5, cap: 6, [98,97,114,49,50]
string():
"bar12"
[0]:
98 = 0x62
[1]:
97 = 0x61
[2]:
114 = 0x72
[3]:
49 = 0x31
[4]:
50 = 0x32
```

I.e `bar12` is coming back as a byte-slice, and not its base 64 encoding `YmFyMTI=`.

Is this as expected?

Reply:
For completeness, I also inspected the next result which isn't human readable and also can attest that the response is not base64 encoded (this time inspecting the response in the rest-client test directly):

```
github.com/algorand/go-algorand/daemon/algod/api/server/v2/generated.BoxResponse {Name: []uint8 len: 8, cap: 9, [0,0,0,0,0,0,48,33], Value: []uint8 len: 5, cap: 6, [0,1,254,3,2]}
Name:
[]uint8 len: 8, cap: 9, [0,0,0,0,0,0,48,33]
string():
"\x00\x00\x00\x00\x00\x000!"
[0]:
0 = 0x0
[1]:
0 = 0x0
[2]:
0 = 0x0
[3]:
0 = 0x0
[4]:
0 = 0x0
[5]:
0 = 0x0
[6]:
48 = 0x30
[7]:
33 = 0x21
Value:
[]uint8 len: 5, cap: 6, [0,1,254,3,2]
string():
"\x00\x01\xfe\x03\x02"
[0]:
0 = 0x0
[1]:
1 = 0x1
[2]:
254 = 0xfe
[3]:
3 = 0x3
[4]:
2 = 0x2
```

Reply:
Great explanation!

Resolving